src/Form/Form.js

```diff
@@ -0,0 +1,64 @@
+import "./Form.css";
+import React, { useState } from "react";
+
+export default function Form() {
+  const [meme, setMeme] = React.useState({
+    topText: "",
+    bottomText: "",
+    randomImage: "http://i.imgflip.com/1bij.jpg",
+  });
+  const [allMemes, setAllMemes] = React.useState([]);
+  React.useEffect(() => {
+    fetch("https://api.imgflip.com/get_memes")
+      .then((res) => res.json())
+      .then((data) => setAllMemes(data.data.memes));
+  }, []);
+  function getMemeImage() {
+    const randomNumber = Math.floor(Math.random() * allMemes.length);
+    const url = allMemes[randomNumber].url;
+
```

Comment:
- [ ]  Include some spacing and group related code lines. for improved readability